dpgen/generator/run.py

```diff
@@ -1676,9 +1813,14 @@ def make_fp_gaussian(iter_index,
     else:
         fp_params = jdata['fp_params']
     cwd = os.getcwd()
+
+    model_devi_engine = jdata.get('model_devi_engine', 'lammps')
     for ii in fp_tasks:
         os.chdir(ii)
-        sys_data = dpdata.System('POSCAR').data
+        if model_devi_engine == "lammps":
+            sys_data = dpdata.System('POSCAR').data
+        elif model_devi_engine == "gromacs":
```

Comment:
Is it designed to only support gaussian?

Reply:
By the way, do you think it's a mass here? We should try to merge the same codes of different fp styles.

Reply:
I think it's not only supported for Gaussian.